API.md

```diff
@@ -1863,6 +1883,8 @@ If there was an error, expect an HTTP status code in either the 4XX or 5XX range
 
 Changes a user's password.
 
+This operation requires rodsadmin level privileges.
```

Comment:
Can a user change their own password by this or any other means in the HTTP API?

Reply:
Not at this time.